### DISCUSSION TITLE
New GitHub feature: Discussions!

### DISCUSSION THREAD

**Discussion by dominiquesydow** [volkamerlab/opencadd#61] [General]
Hi @jaimergp, 

I toggled on GitHub's new Discussions feature for the `opencadd` repo. 
This might be a great way to discuss new or existing modules - and if the discussion leads to something more concrete, we can open an issue.

**Comment by jaimergp** [comment]
Nice! I still don't understand the difference between discussions and issues but happy to explore. Might be interesting for more user-oriented projects like TeachOpenCADD  too!
